services/ui/ws/platform_display_default.cc

```diff
@@ -162,8 +166,8 @@ void PlatformDisplayDefault::UpdateEventRootLocation(ui::LocatedEvent* event) {
 }
 
 void PlatformDisplayDefault::OnBoundsChanged(const gfx::Rect& new_bounds) {
-  // We only care if the window size has changed.
-  if (new_bounds.size() == metrics_.bounds_in_pixels.size())
+  // We only care if the window size has changed and |image_cursors_| exists.
+  if (new_bounds == metrics_.bounds_in_pixels && !image_cursors_)
```

Comment:
yeah, right. this is not needed at all. The call ends up in PlatformDisplayDefault::SetCursorById.
And there is such a check already.